hazelcast-jet-core/src/main/java/com/hazelcast/jet/core/DAG.java

```diff
@@ -363,7 +361,7 @@ public JsonObject toJson(int defaultLocalParallelism) {
             edge.add("to", e.getDestName());
             edge.add("toOrdinal", e.getDestOrdinal());
             edge.add("priority", e.getPriority());
-            edge.add("distributed", e.isDistributed());
+            edge.add("distributedTo", e.getDistributedTo() + "");
```

Comment:
`String.valueOf(e.getDistributedTo())` is nicer than this "append empty string" idiom.